### DISCUSSION TITLE
Failing to deploy a new static project

### DISCUSSION THREAD

**Discussion by av** [vercel/vercel#5109] [Help] [answered]
Having a static project of a single `index.html`:
```html
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Document</title>
</head>
<body>
  <h1>Static page</h1>
</body>
</html>
```
Running `vercel` in Project dir and going through all configuration steps produces following error:
```bash
Error! Invalid vercel.json - `projectSettings.sourceFilesOutsideRootDirectory` Invalid request: `projectSettings.sourceFilesOutsideRootDirectory` should be boolean..
View Documentation: https://vercel.com/docs/configuration#project/projectsettings.sourcefilesoutsiderootdirectory
```
Issue was reproduced with Vercel CLI v20 and v19. Deployments of already existing projects and projects created through GitLab/GitHub integrations are working as expected.

Unfortunately, I haven't been able to locate any references to this config option, looking through Vercel's GitHub org repos.

Have anybody else experienced such an issue? I wonder if there's any way to affect this behavior from Vercel CLI side, but can't find any relevant information.

**Comment by karakanb** [comment]
I have started having this in the last hour or so as well.

**Comment by dodgeblaster** [comment]
Same. Tried single index.html, tried `vc init` nextjs app, tried `/api/index.js` hello world. Getting the same error for all of them.

**Comment by av** [comment, marked as answer]
As an update, I was just able to successfully run a deployment of a new static project. No changes were made on the side of the project or CLI. Issue seem to be gone.
